llvm/test/CodeGen/NVPTX/i128.ll

```diff
@@ -0,0 +1,643 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
I assume that what we're testing for here is the lack of libcalls, not the particular sequence of instructions we generate for i128 divisions. If that's the case then a negative check verifying absence of calls may be a more suitable approach.

Reply:
I think it does both, since we do probably want to verify that it generates the appropriate shifts in PTX.

Reply:
For simple ops, sure. For the real division the code is a bit too large to be a good test one could reason about.

It does the job. If it does not cause too much churn because LLVM rearranges those instructions too often due to unrelated changes, I'm OK with it.